<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>666</number>
   </property>
   <property name="value">
    <number>29</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_A">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>666</number>
   </property>
   <property name="value">
    <number>33</number>
   </property>
   <property name="sliderPosition">
    <number>33</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Шаг аппроксимации</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>параметр кривой</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_B">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>666</number>
   </property>
   <property name="sliderPosition">
    <number>77</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Widget</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_A</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Widget</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Widget</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_B</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Widget</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
